ui/jetbrains/src/main/java/com/neueda/jetbrains/plugin/graphdb/jetbrains/ui/console/params/ParametersService.java

```diff
@@ -43,13 +40,18 @@ public void registerParametersProvider(final ParametersProvider parametersProvid
     }
 
     public Map<String, Object> getParameters(PsiElement element) throws Exception {
-        if (!isValidParametersMap(parametersProvider.getParametersJson())) {
-            return Collections.emptyMap();
+        if (element == null) return Collections.emptyMap();
```

Comment:
Use curly braces

Reply:
Fixed in [b481123](https://github.com/neueda/jetbrains-plugin-graph-database-support/pull/107/commits/b481123a4b4a9620b825b946d8ac783aa1d0c49f)